scoap3/utils/tools.py

```diff
@@ -110,12 +110,22 @@ def author_export(search_year, search_country):
     if search_year:
         search = search.filter("match", publication_date=f"{search_year}-01-01||/y")
 
-    if search_country:
```

Comment:
i think we should keep this, and then display only the authors from the `search_country` for each article returned